pkg/blob/nodeserver.go

```diff
@@ -575,3 +582,24 @@ func (d *Driver) ensureMountPoint(target string, perm os.FileMode) (bool, error)
 	}
 	return !notMnt, nil
 }
+
+func (d *Driver) waitForMount(path string, intervel, timeout time.Duration) error {
+	timeAfter := time.After(timeout)
+	timeTick := time.Tick(intervel)
+
+	for {
+		select {
+		case <-timeTick:
+			notMount, err := d.mounter.IsLikelyNotMountPoint(path)
```

Comment:
LGTM in general, is it possible to add a bind mount to local disk path unit test?

Reply:
mount operation needs root permission, which will cause unit test failed locally.

Reply:
what about writing a ut case for normal path, it should retrun `true, nil`

Reply:
`IsLikelyNotMountPoint` and `mountinfo.MountedFast` are only support for unix, so the ut in windows and mac os system will fail or cannot compile, so I have to use [fake mounter](https://github.com/kubernetes-sigs/blob-csi-driver/pull/852/files#diff-2ac9427651edf35d972cc03bfea020855df0d4b212b619588756a71d8d875ee9R804).

Reply:
I think we could skip windows and mac ut if it does not work, that's ok. It's better to have such test coverage since that's quite fundamental change.

Reply:
fxied, please take another look.

Reply:
thanks, pls squash commits